[bug] Autoware on Melodic fails to build: calibration_publisher. undefined reference to `cv::read`

# Bug
Cannot compile `calibration_publisher` on Melodic #2032 

```
--- stderr: calibration_publisher                                                                        
CMakeFiles/calibration_publisher.dir/src/calibration_publisher.cpp.o: In function `void cv::operator>><std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >(cv::FileNode const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&)':
calibration_publisher.cpp:(.text+0x40): undefined reference to `cv::read(cv::FileNode const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
collect2: error: ld returned 1 exit status
```

## Specifications

  - Ubuntu version: 18.04
  - ROS version: Melodic
  - Autoware branch: feature/melodic

--- a/ros/src/sensing/fusion/packages/calibration_publisher/CMakeLists.txt
+++ b/ros/src/sensing/fusion/packages/calibration_publisher/CMakeLists.txt
@@ -43,6 +43,7 @@ add_dependencies(calibration_publisher
 
 target_link_libraries(calibration_publisher
         ${catkin_LIBRARIES}
+        ${OpenCV_LIBS}
         )
 
 install(TARGETS calibration_publisher

--- a/ros/src/sensing/fusion/packages/calibration_publisher/package.xml
+++ b/ros/src/sensing/fusion/packages/calibration_publisher/package.xml
@@ -19,5 +19,6 @@
   <depend>cv_bridge</depend>
   <depend>image_transport</depend>
   <depend>tf</depend>
+  <depend>libopencv-dev</depend>
 
 </package>

